### PR TITLE
#18

### PR DESCRIPTION
**Pull request by Pshpdesu**
Tried to rework some stuff and fix problem #18

### CODE REVIEW

File: PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs [hunk 1]
```diff
@@ -44,79 +43,68 @@ public Win_WorkImgs()
...
         }
+        Dictionary<ImageUrls, Image> urlImages;
         IllustWork work;
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
             base.OnNavigatedTo(e);
             work = e.Parameter as IllustWork;
-            flipview.ItemsSource = work.meta_pages;
             flipview.SelectedIndex = 0;
+            urlImages = new Dictionary<ImageUrls, Image>();
```
[the remark this answers is not in the document]
Comment (Pshpdesu) [reply PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs]: Maybe bitmaps instead of Image?

[the remark this answers is not in the document]
Comment (Pshpdesu) [reply PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs]: And now I think, what using just ImageUrls also not a great idea 🤔

[the remark this answers is not in the document]
Comment (Pshpdesu) [reply PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs]: Maybe find image number from url in GetHashCode of ImageUrls?

[the remark this answers is not in the document]
Comment (yinyue200) [reply PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs]: maybe using bitmap as value and ImageUrls as key is ok.

[the remark this answers is not in the document]
Comment (Pshpdesu) [reply PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs]: Why Image is not ok in this case?

[the remark this answers is not in the document]
Comment (yinyue200) [reply PixivUWP/Pages/DetailPage/Win_WorkImgs.xaml.cs]: Image is a ui control. I think we shoudn't use it in this case even if it will work.

**Pull request closed without merging**